modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -28,7 +28,7 @@ image::odc_customizing_developer_catalog.png[]
 
 [NOTE]
 ====
-As an administrator, you can define the navigation items that appeaer by default for all users. You can also reorder the navigation items.
+As an administrator, you can define the navigation items that appear by default for all users. You can also reorder the navigation items.
```

Comment:
**Suggestion**: Use admonitions sparingly so that they do not lose their effectiveness. Refer to this guideline [here](https://stylepedia.net/style/6.0/#admonitions). 

- Move the Note section up, so that it appears immediately after the step 4 instruction and just before the Verification section.